server/apps/prepopulate/app_initialize.py

```diff
@@ -167,4 +171,21 @@ def import_file(self, entity_name, file_name, index_params, do_patch=False):
                 self.logger.info('Index: {} for collection {} created successfully.'.format(index_name, entity_name))
 
 
+def fillEnvironmentVariables(config, item):
+    variables = {}
+    text = json.dumps(item)
+
+    for variable in findall('#ENV_([^#"]+)#', text):
+        value = config.get(variable, None)
```

Comment:
maybe it could read it from `env` so that we don't have to add it into settings for every var we want to use?